plugins/colorbutton/plugin.js

```diff
@@ -384,8 +384,17 @@ CKEDITOR.plugins.add( 'colorbutton', {
 
 			return output.join( '' );
 
-			function setColor( color ) {
-				var colorStyle = color && new CKEDITOR.style( colorStyleTemplate, { color: color } );
+			function setColor( color, colorName ) {
+				var colorStyleVars = {};
+
+				if ( color ) {
+					colorStyleVars.color = color;
+				}
+				if ( colorName ) {
+					colorStyleVars.colorName = colorName;
+				}
+
+				var colorStyle = color && new CKEDITOR.style( colorStyleTemplate, colorStyleVars );
```

Comment:
Looks like a leftover, checking for `color` doesn't make sense after recent changes above. Maybe something like [`isEmpty()`](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_tools.html#method-isEmpty)?